util.go

```diff
@@ -110,8 +110,34 @@ func parseDenomFromJson(tx []byte) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	body := anyJson["body"].(map[string]interface{})
-	denom := body["messages"].([]interface{})[0].(map[string]interface{})["amount"].([]interface{})[0].(map[string]interface{})["denom"].(string)
-	return denom, nil
+	authJson := anyJson["auth_info"]
+	if authJson != nil {
+		auth := authJson.(map[string]interface{})
```

Comment:
I think the canonical check for typecasting in Go is something like this. But if this didn't panic, it's a good enough fix for now.

```
		if auth, ok := authJson.(map[string]interface{}); ok {
                  // auth is available to use
                }
```